model/Core/Properties/validUntilTime.md

```diff
@@ -0,0 +1,18 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# validUntilTime
+
+## Summary
+
+Specifies the time until which an artifact is valid.
+
+## Description
+
+A validUntilTime specifies the time until which an artifact is valid.
```

Comment:
Replace this with something closer to the semantics in 2.3. 

Suggest: 
" A valdiUntilTime specifies when the artifact can be used until it's useage needs to be reassessed"

Reply:
fixed